### ISSUE TITLE
Update ts codegen tooling

### ISSUE THREAD

**Issue by ethanfrey** [osmosis-labs/beaker#97]
@pyramation just released `cosmwasm-typescript-gen` as `@cosmwasm/ts-codegen`

Let's use that now.

`beaker wasm ts-gen xyz` leads to:

```
Need to install the following packages:
  cosmwasm-typescript-gen
```

**Comment by iboss-ptk** [comment]
blocked by https://github.com/CosmWasm/ts-codegen/issues/52

**Comment by pyramation** [comment]
should be ready!

**Comment by iboss-ptk** [comment]
resolved by #100
